bcda/main.go

```diff
@@ -483,8 +483,7 @@ func createAlphaToken(ttl int, acoSize string) (s string, err error) {
 		result = fmt.Sprintf("%s\n%s\n%s", expiresOn, tokenId, token.TokenString)
 
 	case auth.OktaAuthPlugin:
-		expiresOn := time.Now().AddDate(1, 0, 0).Format(time.RFC850)
-		result = fmt.Sprintf("%s\n%s\n%s", expiresOn, creds.ClientID, creds.ClientSecret)
+		result = fmt.Sprintf("%s\n%s\n%s", creds.ClientName, creds.ClientID, creds.ClientSecret)
```

Comment:
Is it time yet to change the name of `createAlphaToken`?

Reply:
I think we want to leave that to when we actually the alpha impl to return credentials.